src/test/java/com/epam/ShoppingCart.java

```diff
@@ -0,0 +1,33 @@
+package com.epam;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class ShoppingCart {
+
+    @Test
+    @DisplayName("Verify shopping cart is empty & summary steps are displayed")
+
+    public void dressesCount() {
+        System.setProperty("webdriver.chrome.driver", "chromedriver.exe");
+        WebDriver driver = new ChromeDriver();
+        driver.get("http://automationpractice.com/index.php");
+
+        driver.findElement(By.xpath("//div/a[@title='View my shopping cart']")).click();
```

Comment:
in the current context - as the site is single-language - that's working, but still more prone to changes.
`//*[@class='shopping_cart']` is also unique and seems a bit more stable.

most of the times, the parent element of an anchor has the same display area and clicking on them will also work for their children. but, not all the times.

as an even safer solution would be to use something like this:
`//*[@class='shopping_cart']/a` and/or utilize the href partial check of the link.

Reply:
Totally agree. Wil try to address in future.